tests/unit/lms/services/user_test.py

```diff
@@ -143,13 +143,29 @@ def test_get_users_by_course_id(
             admin_organization_ids=[organization.id],
         )
 
-        assert db_session.scalars(query).all() == [student_in_assigment]
+        assert db_session.scalars(query).all() == [student_in_assignment]
+
+    @pytest.mark.usefixtures("assignment")
+    @pytest.mark.parametrize("with_h_userids", [True, False])
+    def test_get_users_for_course(
```

Comment:
There are new tests for the new methods here, the older method is left untouched.